yarn.lock

```diff
@@ -25433,6 +25499,14 @@ string.prototype.trimstart@^1.0.1:
     define-properties "^1.1.3"
     es-abstract "^1.18.0-next.1"
 
+string.prototype.trimstart@^1.0.3:
```

Comment:
This should be deduplicated -- now we install both v1.0.1 and v1.0.3

Reply:
Good catch!